packages/controllers/src/snaps/SnapController.ts

```diff
@@ -108,6 +108,25 @@ export type Snap = {
   version: string;
 };
 
+export interface SnapRuntimeData {
```

Comment:
This could use a docstring explaining why it exists and why we don't keep it in state.

Reply:
Done!